package.json

```diff
@@ -9,6 +9,10 @@
   "author": "Mohamed Atef",
   "license": "ISC",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
     "webpack": "^5.28.0",
     "webpack-cli": "^4.5.0"
   }
```

Comment:
**REQUIRED**
Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png))
```js
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```